frege/compiler/passes/Instances.fr

```diff
@@ -263,8 +263,8 @@ deriveClass pos clas forty ctrs instrho g toderive = deriveClass toderive
                 alt1      = calt vEq cas
                 alt2      = calt (var "x") (var "x")
                 cas       = if isEnum then vEq else Case CNormal (vtup varg1 varg2) eqexs
-                eqexs     = (map mkeqcase ctrs) ++ [defalt]
-                defalt    = calt (var "_") vEq
+                eqexs     = (map mkeqcase ctrs) ++ [default]
```

Comment:
This will have to be changed to something other than `default` which is apparently a reserved word (afaict from the compiler output).